tests/test_dumpers.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Module tests."""
+
+from invenio_db import db
+from invenio_records.dumpers import ElasticsearchDumper
+
+from invenio_rdm_records.api import BibliographicRecord
+from invenio_rdm_records.dumpers import EDTFDumperExt
+
+
+def test_esdumper_with_edtfext(app, db, minimal_input_record):
+    """Test edft extension implementation."""
+    # Create a simple extension that adds a computed field.
+
+    dumper = ElasticsearchDumper(extensions=[EDTFDumperExt()])
+
+    minimal_input_record['metadata']['publication_date'] = '2021-01-01'
+
+    # Create the record
+    record = BibliographicRecord.create(minimal_input_record)
+    db.session.commit()
+
+    # Dump it
+    dump = record.dumps(dumper=dumper)
+    assert dump['metadata']['publication_date_start'] == '2021-01-01'
```

Comment:
Comment: I would also assert ``publication_date`` to ensure it's not removed by the dumper.